lib/validators/movies/list.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const Joi = require('joi');
+
+module.exports = Joi.object().keys({
+  title: Joi.string().min(1).max(255).optional(),
```

Comment:
ya i think @tonywuhl is correct, we don't want the API to change despite the column name changing. I think that's the point of the zero-downtime migration section. if its confusing, lmk and we can work on re-wording it.